ui/src/app/base/components/FormikForm/FormikForm.tsx

```diff
@@ -53,6 +54,7 @@ const FormikForm = <V, E = null>({
         cancelDisabled={cancelDisabled}
         className={className}
         cleanup={cleanup}
+        data-testid={dataTestId}
```

Comment:
```suggestion
```
It might be best not to encourage the use of test id when we have a better alternative.
In this case we can easily use an aria-label in combination with a semantic role in tests, e.g.:

```jsx
getByRole("form", { name: "Edit VLAN" })
```

Reply:
Thanks, now that I've figured out how to set `role="form"` correctly (see my other comment) I can use the label properly.